[core]/es_extended/server/classes/player.lua

```diff
@@ -572,8 +572,8 @@ function CreateExtendedPlayer(playerId, identifier, group, accounts, inventory,
 		end
 	end
 
-	function self.showNotification(msg)
-		self.triggerEvent('esx:showNotification', msg)
+	function self.showNotification(msg, type, length)
```

Comment:
Hi @Gellipapa, thanks for reviewing the code.

As this is a wrapper function like xPlayer.showAdvancedNotification, the default values are handled by the actual function that gets called on the client. In this case it is handled by esx_notify.

As for the parameter name, I looked at the parameters of the client event en function to have the exact same parameters in this wrapper function as there. In the event and client function the `type` parameter there is also called type. The same name is used for ESX.TextUI. I believe that in this case it is more important to have consistency between all functions than to avoid reassigning `type`, as a wrapper function that should be identical to the client function would otherwise not make much sense. If you do want it to be `notificationType`, I think that all parameters named `type` should be renamed first to ensure consistency.

Let me know if you still need me to change it.